Cannot find module 'recompose/compose'.

## Steps to Reproduce

Just import `react-apollo`.

### Buggy Behavior

```
[0] /home/zhenghu/coding/qsc/qsc-main-react-typescript/node_modules/react-apollo/index.d.ts
[0] (3,21): error TS2307: Cannot find module 'recompose/compose'.
```

### Expected Behavior

No error.

### Version
- apollo-client@0.4.21
- react-apollo@0.5.11

--- a/Changelog.md
+++ b/Changelog.md
@@ -4,6 +4,8 @@ Expect active development and potentially significant breaking changes in the `0
 
 ### vNext
 
+- Bug: Fix issue with usage in TypeScript projects caused by 'compose' re-export. [PR #291](https://github.com/apollostack/react-apollo/pull/291)
+
 ### v0.5.12
 
 - Full support for both Apollo Client 0.4.21 and 0.5.0. [PR #277](https://github.com/apollostack/react-apollo/pull/277)

--- a/package.json
+++ b/package.json
@@ -115,7 +115,6 @@
     "lodash.isequal": "^4.1.1",
     "lodash.isobject": "^3.0.2",
     "lodash.pick": "^4.4.0",
-    "object-assign": "^4.0.1",
-    "recompose": "^0.20.2"
+    "object-assign": "^4.0.1"
   }
 }

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,7 +1,7 @@
 import ApolloProvider from './ApolloProvider';
 import graphql, { withApollo } from './graphql';
 
-// expose easy way to join queries from recompose
-import compose from 'recompose/compose';
+// expose easy way to join queries from redux
+import { compose } from 'redux';
 
 export { ApolloProvider, graphql, withApollo, compose };

--- a/typings.d.ts
+++ b/typings.d.ts
@@ -24,11 +24,6 @@ declare module 'lodash.pick' {
   export = main;
 }
 
-declare module 'recompose/compose' {
-  function hoc(component: any): any;
-  export default (...hocs) => hoc;
-}
-
 declare module 'hoist-non-react-statics' {
   /**
    * Copies any static properties present on `source` to `target`, excluding those that are specific

